src/components/calculator.js

```diff
@@ -1,29 +1,50 @@
 import React from 'react';
+import calculate from '../logic/calculate';
+
+class Calculator extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      total: '0',
+      next: null,
+      operation: null,
+    };
+    this.clickHandle = this.clickHandle.bind(this);
+  }
+
+  clickHandle = (event) => this.setState((prevState) => {
+    calculate(prevState, event.target.innerHTML);
+  });
```

Comment:
**Changes Required**
- [ ] `Calculator.js` is a child component of `App.js`  Props need to get passed to child components of the Calculate.js component. You need to make `App.js` a stateful component and then pass down the state to it's children using `props`.
- [ ] The `clickHandle` function currently does not work and the calculations are not being displayed on the web app. Let us try to fix this bug.

Reply:
Required changes satisfied.